GAE/src/com/gallatinsystems/survey/dao/CaddisflyResourceDao.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (C) 2014 Stichting Akvo (Akvo Foundation)
```

Comment:
year: fixed
I've turned on formatting now - it was not being done automatically.